tools/validators/ontology_validator/yamlformat/tests/presubmit_validate_types_lib_test.py

```diff
@@ -228,6 +228,31 @@ def testConfigUniverseGetUnitsForMeasurement(self):
         'discharge_fan_lost_power_alarm')
     self.assertIsNone(units)
 
+  def testConfigUniverseGetUnitsForMeasurementMultipleNoUnits(self):
+    doc = {
+      'powerfactor': [
+        {'no_units': 'STANDARD'},
+        'another_one'
+      ],
+      'voltageratio': [{'no_units': 'STANDARD'}],
+    }
+    folder = unit_lib.UnitFolder('units/anyfolder')
+    folder.AddFromConfig([doc], 'units/anyfolder/units.yaml')
+    unit_universe = unit_lib.UnitUniverse([folder])
+
+    config_universe = presubmit_validate_types_lib.ConfigUniverse(
+        subfield_universe=None,
+        field_universe=None,
+        entity_type_universe=None,
+        state_universe=None,
+        connection_universe=None,
+        unit_universe=unit_universe)
+
+    units = config_universe.GetUnitsForMeasurement('powerfactor_sensor')
```

Comment:
line 251 to move up with the previous config_universe block
then leave empty blank line before self.assertSae...